src/App.js

```diff
@@ -0,0 +1,41 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { persistStore } from 'redux-persist';
+import { PersistGate } from 'redux-persist/integration/react';
+import SignUp from './components/SignUp';
+import store from './redux/store';
+import Login from './components/Login';
+import Doctor from './components/Doctor';
+import Appointment from './components/Appointment';
+import AppointmentListing from './components/AppointmentListing';
+import Dashboard from './Dashboard';
+import DoctorList from './components/DoctorList';
+import DoctorDetails from './components/DoctorDetails';
+
+const persistor = persistStore(store);
+
+function App() {
```

Comment:
- [ ] Please, kindly refactor this and all other functional components/functions on your project to ES6 arrow function, so your code complies with the latests standards.

The function keyword has been soft-deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non-best practice.

Refactoring  functions to ES6 is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can do a search in Vscode for the "function" word, and correct as needed I found these:  

![image](https://user-images.githubusercontent.com/57421823/203661202-bd7834e5-144b-4d0d-b402-e8f0418f330f.png)


It shouldn't take you more than 5-10 min so don't worry.